internal/utils/templates/initial_schemas/15.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 15.1 (Debian 15.1-1.pgdg110+1)
--- Dumped by pg_dump version 15.0
+-- Dumped from database version 14.5 (Debian 14.5-2.pgdg110+2)
+-- Dumped by pg_dump version 14.7 (Homebrew)
```

Comment:
Seems like the dump is created from pg14 database. Could you double check the major version in `supabase/config.toml`?

Reply:
Ah yeah my bad, have just changed it, pushing up the changes now